hms-app/src/components/card-details/IdeaDetails.tsx

```diff
@@ -108,17 +115,19 @@ export default function IdeaDetails(props: IProps) {
       opened={editModalOpened}
       onClose={() => setEditModalOpened(false)}
       withCloseButton={false}
+      size="55%"
     >
       Edit Idea
       <IdeaForm
         ideaID={idea.id}
+        idea={idea}
         context={'edit'}
-        userId={idea.owner?.id!}
+        participantID={idea.owner?.id!}
         hackathon={idea.hackathon!}
+        setOpened={closeEditModal}
```

Comment:
I think you don't need to introduce `closeEditModal` as `setEditModalOpened` is pretty self explanatory
```suggestion
        setOpened={setEditModalOpened}
```